core/src/test/kotlin/io/specmatic/core/CalculatePathTest.kt

```diff
@@ -54,7 +54,7 @@ internal class CalculatePathTest {
 
         val paths = pattern.calculatePath(value, Resolver())
         
-        assertThat(paths).containsExactly("value")
+        assertThat(paths).containsExactly("value{number}")
```

Comment:
In this test, there is an object with no typeAlias at the top level. Write a test for an array at the top level. The relevant paths should be in the style of "[0]{string}" or something like it depending on which element in the array and type the value turned out to be.